backend/src/migrations/20190729000000-initialize-database.js

```diff
@@ -0,0 +1,228 @@
+          userId: {
+            allowNull: false,
+            defaultValue: Sequelize.UUIDV4,
+            primaryKey: true,
+            type: Sequelize.UUID
+          },
+          createdAt: {
+            allowNull: false,
+            type: Sequelize.DATE
+          },
+          updatedAt: {
+            allowNull: false,
+            type: Sequelize.DATE
+          },
+          deletedAt: {
+            allowNull: true,
+            type: Sequelize.DATE
+          },
+          keycloakId: {
+            allowNull: false,
+            comment: 'Associated Keycloak user id',
+            type: Sequelize.UUID,
+            unique: true
+          },
+          displayName: {
+            allowNull: false,
+            comment: 'Display name of the user',
+            type: Sequelize.STRING(32)
+          }
+        }, {
+          comment: 'List of all registered users',
+          transaction: t
+        }),
+        queryInterface.createTable('user_acronym', {
+          userAcronymId: {
+            allowNull: false,
+            defaultValue: Sequelize.UUIDV4,
+            primaryKey: true,
+            type: Sequelize.UUID
+          },
+          createdAt: {
+            allowNull: false,
+            type: Sequelize.DATE
+          },
+          updatedAt: {
+            allowNull: false,
+            type: Sequelize.DATE
+          },
+          deletedAt: {
+            allowNull: true,
+            type: Sequelize.DATE
+          },
+          userId: {
+            allowNull: false,
+            references: {
+              model: 'user',
+              key: 'userId'
+            },
+            type: Sequelize.UUID
+          },
+          acronymId: {
+            allowNull: false,
+            references: {
+              model: 'acronym',
+              key: 'acronymId'
+            },
+            type: Sequelize.UUID
+          },
+          owner: {
+            allowNull: false,
+            comment: 'Does user have ownership privileges on acronym',
+            defaultValue: false,
+            type: Sequelize.BOOLEAN
+          }
+        }, {
+          comment: 'Correlation between users and acronyms',
+          transaction: t
+        }),
+        queryInterface.createTable('lifecycle', {
+          lifecycleId: {
+            allowNull: false,
+            defaultValue: Sequelize.UUIDV4,
+            primaryKey: true,
+            type: Sequelize.UUID
+          },
+          createdAt: {
+            allowNull: false,
+            type: Sequelize.DATE
+          },
+          updatedAt: {
+            allowNull: false,
+            type: Sequelize.DATE
+          },
+          deletedAt: {
+            allowNull: true,
+            type: Sequelize.DATE
+          },
+          acronymId: {
+            allowNull: false,
+            references: {
+              model: 'acronym',
+              key: 'acronymId'
+            },
+            type: Sequelize.UUID
+          },
+          appCfg: {
```

Comment:
Should use a more easily understood name for this field...  If all the node code is using appCfg to call the WEBADE Rest API, then the class property name is ok, but the database column name should probably be changed to application_config or something clear.  Even app_config is better than 2 abbreviations

Reply:
I've gone ahead and changed it to appConfig - there's a lot existing code calling it as appConfig.